### PR TITLE
Update LinearWeightedMovingAverage.cs

### PR DESCRIPTION
**Pull request by mr-september**
Fix off by one indexing error in denominator, and calculation error in numerator.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] Performance (non-breaking change which improves performance. Please add associated performance test and results)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [ ] I have added tests to cover my changes. 
- [ ] All new and existing tests passed.
- [ ] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

**Comment by Martin-Molinero** [comment]
Hey @mr-september! Welcome to Lean
Please share some information about the issue, error/how to reproduce? We should add some unit test reproducing the issue so we can assert it's nature and how to and if it's been fixed 👍

File: Indicators/LinearWeightedMovingAverage.cs [hunk 1]
```diff
@@ -77,7 +77,7 @@ protected override decimal ComputeNextValue(IReadOnlyWindow<IndicatorDataPoint>
 
             // The denominator is calculated each time in case the Size is less than the period.
             // There may be a more efficient way of calculating the factorial.
-            for (var i = 0; i <= index; i++)
+            for (var i = 0; i < index; i++)
```
Comment (scloudyy) [inline Indicators/LinearWeightedMovingAverage.cs:80]: why not `var denominator = index`?

**Comment by mr-september** [comment]
> Hey @mr-september! Welcome to Lean Please share some information about the issue, error/how to reproduce? We should add some unit test reproducing the issue so we can assert it's nature and how to and if it's been fixed 👍

Hi, my apologies, I am not an experienced developer, especially not with C#. I have merely noticed inconsistent behaviour while porting strategies from another platform to QuantConnect, and after some attempts to investigate, I believe this may be the root of the issue.

It is not a technical error, but a mathematical/logical one.

File: Indicators/LinearWeightedMovingAverage.cs (hunk 1 above)
Comment (mr-september) [reply Indicators/LinearWeightedMovingAverage.cs:80]: Hi, if you mean this as a technical suggestion, then I am unable to comment. Your suggestion is most likely better. 

I am only trying to supply the smallest, most compatible change to what I believe may be a mathematical/logical error in the implementation of this indicator.

**Comment by Marinovsky** [comment]
Thank you @mr-september, we've revised the current implementation and believe it's correct, we've added more tests comparing the expected values with 3rd party.

**Pull request closed without merging**